tools/integration_tests/mounting/test.txt

```diff
@@ -0,0 +1,24 @@
+{"name":"root","levelname":"INFO","severity":"INFO","message":"Start gcsfuse/fake_version (Go version go1.19-pre4 cl/455575533 +12f49fe0ed) for app \"\" using mount point: /tmp/gcsfuse_test1743820428\n","timestampSeconds":1662540221,"timestampNanos":695732107}
```

Comment:
I think this file should be deleted.
@raj-prince I think these files are created as a side effect of running go test all in the gcsfuse repo. Can we make sure it gets deleted after the tests are completed? Or make this in a temporary directory?